tools/nl/embeddings/build_custom_dc_embeddings.py

```diff
@@ -24,6 +25,15 @@
 import utils
 import yaml
 
+# Import gcs module from shared lib.
+# Since this tool is run standalone from this directory,
+# the shared lib directory needs to be appended to the sys path.
+_THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+_SHARED_LIB_DIR = os.path.join(_THIS_DIR, "..", "..", "..", "shared", "lib")
```

Comment:
nit: why not "../../../shared/lib"

Reply:
Just a good practice to let the os module insert the system path separator ("/" or "\") instead of hardcoding it.

Reply:
I see.  Readability is poor, but sure :)